scripts/metadata-export.md

```diff
@@ -0,0 +1,42 @@
+# Metadata-Export script
+<!-- TOC -->
+* [Metadata-Export script](#metadata-export-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Metadata-Export script exports the metadata for a specific item, collection, or community in CSV format.
+
+## When we use it?
+
+The Metadata-Export script is used to export metadata for batch editing in DSpace. Users can export metadata for
+specific items, collections, or communities, and then modify the metadata in a spreadsheet or text editor. Once the
+metadata has been edited, it can be re-imported using the Metadata-Import script.
```

Comment:
I'd recommend we link in the Batch Metadata Editing docs, as that has import info about the format of the CSV.

"For more information see the [Batch Metadata Editing](https://wiki.lyrasis.org/display/DSDOC7x/Batch+Metadata+Editing) documentation"